lib/12T/lef/gf180mcu_osu_sc_12T.lef

```diff
@@ -0,0 +1,2661 @@
+VERSION 5.7 ;
```

Comment:
is the lef generated? (if yes maybe we could remove it from the repo and document how to build it from other source files)

Reply:
I don't understand this comment.  You need the LEF and you cannot generate it automatically.   I think its too premature to have instructions on how to generate, as well.

Reply:
should we add an overview in the README.md then, that highlight which files are generated and which one are hand-authored, with some brief description of the process to maintain them? (otherwise it might be challenging for others to contribute to the repo, ex: they might start modifying manually things that are generated)